commitlint/helpers.ts

```diff
@@ -19,18 +19,28 @@ export abstract class Helpers {
     public static convertAnyToString(
         potentialString: any,
         paramName: string
-    ): string {
+    ): string | null {
         if (potentialString === null || potentialString === undefined) {
             // otherwise, String(null) might give us the stupid string "null"
+            return null;
+        }
+        return String(potentialString);
+    }
+
+    public static assertRaw(rawStr: string | null) {
+        if (rawStr === null) {
             throw new Error(
-                "Unexpected " +
-                    paramName +
-                    "===null or " +
-                    paramName +
-                    "===undefined happened"
+                "This is unexpected because raw should never be null"
+            );
+        }
+    }
+
+    public static assertHeader(headerStr: string | null) {
```

Comment:
@realmarv good work! this PR is almost ready, but I just realised that, if you're using this assertFoo functions, you can improve 2 things here:
- Have a generic single function instead of 2 functions, call it assertNotNull, pass the error message as 2nd argument.
- Have this function return string, and do the cast with `as` inside the function.

Reply:
Done